mim/utils/mmpack/README.md

```diff
@@ -0,0 +1,86 @@
+# Export (experimental)
+
+`mim export` is a new function in `mim`, which can export a minimum trainable and testable model package through the config file.
+
+The minimal model compactly combines `model, datasets, engine` and other components. There is no need to search for each module separately based on the config file. Users who do not install from source can also get the model file at a glance.
+
+In addition, lengthy inheritance relationships, such as `CondInstBboxHead -> FCOSHead -> AnchorFreeHead -> BaseDenseHead -> BaseModule` in `mmdetection`, will be directly flattened into `CondInstBboxHead -> BaseModule`. There is no need to open and compare multiple model files when all functions inherited from the parent class are clearly visible.
+
+### Instructions Usage
+
+```bash
+mim export config_path
+
+# config_path has the following three optional types:
+#Config of downstream repo: Complete the call through repo::configs/xxx.py. For example:
+mim export mmdet::configs/mask_rcnn/mask-rcnn_r101_fpn_1x_coco.py
+
+# config in a certain folder, for example:
+mim export config_dir/mask-rcnn_r101_fpn_1x_coco.py
+```
+
+### Minimum model package directory structure
+
+```
+minimun_package(Named as pack_from_{repo}_20231212_121212)
+|- pack
+| |- configs # Configuration folder
+| | |- model_name
+| | |- xxx.py # Configuration file
+| |
+| |- models # model folder
+| | |- model_file.py
+| | |- ...
+| |
+| |- data # data folder
+| |
+| |- demo # demo folder
+| |
+| |-datasets #Dataset class definition
+| | |- transforms
+| |
+| |- registry.py # Registrar
+|
+|- tools
+| |- train.py # training
+| |- test.py # test
+|
+```
+
+### limit
+
+The implementation of `mim export` function depends on `mim/utils/mmpack`. Currently, only some config files of `mmpose/mmdetection/mmagic/mmsegmentation` are supported. And there are some constraints on the downstream repo.
```

Comment:
We need to clarify which limitations will not be supported and which ones will be improved or supported in the future.

Reply:
Done.